app/src/main/kotlin/com/wire/android/ui/sharing/ImportMediaScreen.kt

```diff
@@ -380,24 +412,36 @@ private fun ImportMediaContent(
                 )
             }
         } else if (!isMultipleImport) {
-            Box(modifier = Modifier.padding(horizontal = dimensions().spacing16x)) {
-                ImportedMediaItemView(
-                    item = importedItemsList.first(),
-                    isMultipleImport = false
+            Box(
+                modifier = Modifier
+                    .padding(horizontal = dimensions().spacing16x)
+                    .height(dimensions().spacing120x)
```

Comment:
Juuust making sure the fixed height dimension doesn't break any device with smaller screen? 👀

Reply:
No, because under it is scrollable conversation list 😄